bot/exts/blurple_formatter.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import traceback
+
+import discord
+from botcore.utils.regex import FORMATTED_CODE_REGEX
+from discord.ext import commands
+
+from bot.bot import SirRobin
+from bot.utils import blurple_formatter
+
+
+class BlurpleFormatter(commands.Cog):
+    """Format code in accordance with PEP 9001."""
+
+    def __init__(self, bot: SirRobin):
+        self.bot = bot
+
+    @commands.command()
+    async def blurplify(self, ctx: commands.Context, *, code: str) -> None:
+        """Format code in accordance with PEP 9001."""
+        if match := FORMATTED_CODE_REGEX.match(code):
+            code = match.group("code")
+        try:
+            blurpified = blurple_formatter.blurplify(code)
```

Comment:
We should be running this in a thread:
```suggestion
            blurpified = await asyncio.to_thread(blurple_formatter.blurplify, code)
```
Don't forget to import `asyncio`